.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Integration Tests
+on:
+  workflow_dispatch
```

Comment:
fine if we start manually, but the idea is to run this automatically every night.

Reply:
Added integration tests to check-for-updates.

Reply:
I've added another job now which requires the nightly build to be run first.